MANIFEST.in

```diff
@@ -1 +1,2 @@
+include *.py
```

Comment:
Why is this line needed? I believe we should be able to specify all needed .py files via setup.py, either via packages or scripts.

Reply:
That's part of a long story. I had a very hard time trying to add the new bgp_*.py files into a show/bgp/ and clear/bgp subfolders. I gave up at the end and i left my new files at the same level as main.py. This added line, reflects part of my effort tweaking packages' building-config to add the new bgp folder to the .deb package. Even though i was able to incorporate the new folders into the generated python-sonic-util.tar.gz, these ones never made it into the python-sonic-util.deb package. I gave up at the end, and i decided to remove the bgp folders, so I will go ahead and remove this line now.